persist/adapter.go

```diff
@@ -65,3 +66,22 @@ type Adapter interface {
 	// This is part of the Auto-Save feature.
 	RemoveFilteredPolicy(sec string, ptype string, fieldIndex int, fieldValues ...string) error
 }
+
+type ContextAdapter interface {
+	Adapter
```

Comment:
This part should create a new file. Refer to [batch_adapter.go](https://github.com/casbin/casbin/blob/master/persist/batch_adapter.go)